lib/console_client/game_ui.rb

```diff
@@ -28,24 +27,32 @@ def show_game_result(game)
     private
 
     def display_board(board)
+      rows = board.get_rows
       @io.display ""
-      draw_row(board[0], board[1], board[2])
-      draw_spacer_row
-      draw_row(board[3], board[4], board[5])
-      draw_spacer_row
-      draw_row(board[6], board[7], board[8])
+      rows.each.with_index do |row, i|
+        draw_row(row, board.size)
+        draw_spacer_row(row, board.size) unless i == rows.length - 1
+      end
     end
 
-    def draw_row(a, b, c)
-      @io.display(" #{a} | #{b} | #{c} ")
+    def draw_row(row, width)
+      r = ""
+      row.each.with_index do |p, i|
+        r += " #{p} ".center(width)
+        r += "|" unless i == row.length - 1
+      end
+      @io.display(r)
```

Comment:
Maybe set these functions up so that they return the string and then IO is just called once with the entire string? So it's not everywhere?